src/core/aws-config.ads

```diff
@@ -178,6 +178,11 @@ package AWS.Config is
    --  directory returned will end with a directory separator.
 
    function Upload_Size_Limit (O : Object) return Positive with Inline;
+   --  Size limit for the client uploading data before calling the user's
+   --  callback or dispatcher handler. User can call
+   --  AWS.Status.Is_Body_Uploaded to understand either client data uploaded or
```

Comment:
...to check if client data is uploaded or not because...

Reply:
I did not see your comment, strange. Updated.